.github/ISSUE_TEMPLATE/new-diag-study-template.md

```diff
@@ -0,0 +1,76 @@
+---
+name: New Study Template
+about: Systematically summarize each therapy-focused paper
+title: 'New Paper: [Title]'
+labels: New Paper
+assignees: ''
+
+---
+
+<!--Hi there! Please use the template below as a guide for what information about this paper to include. It's ok to leave fields blank so that other contributors can fill them in later, or to add fields that you think are important.-->
+
+<!--Title: Please edit the title to add the name of the paper after the colon--!>
+
+### Please paste a link to the paper or a citation here:
+
+### What is the paper's [DOI](https://library.uic.edu/help/article/1966/what-is-a-doi-and-how-do-i-use-them-in-citations)?
+
+### Is this paper primarily relevant to Background, Diagnostics, or Therapeutics? (use separate template for diagnostics vs therapeutics)
+
+### Please list some keywords (3-10) that help identify the relevance of this paper to COVID-19
+
+### Questions to answer about each paper:
```

Comment:
```suggestion

<!-- The following questions are the things that we'll want to answer about each paper reporting a diagnostic. If you are ready to answer them now, please go ahead and interleave your answers with the prompts. If you are not ready, feel free to go ahead and create the issue now. -->

### Questions to answer about each paper:
```

Reply:
I wanted to make it clear that folks didn't necessarily have to answer all of the points below to let us know about a paper.